ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -307,6 +307,20 @@ class SignalSpec: QuickSpec {
 				sendCompleted(observer)
 				expect(completed).to(beTruthy())
 			}
+
+			context("memory") {
+				it("should not crash allocating memory with a few observers") {
+					let (signal, sink) = Signal<Int, NoError>.pipe()
+
+					for i in 0..<50 {
+						autoreleasepool {
+							let disposable = signal.observe { _ in }
```

Comment:
This crashes in the 39th iteration **only** in Swift 2.1 (see [radar](http://www.openradar.me/22873620)) before the workaround.